setup.py

```diff
@@ -28,9 +28,10 @@
     install_requires=[
         "datajoint>=0.13.0",
         "ipykernel>=6.0.1",
+        "scikit-learn==1.4.*",
         "opencv-python",
         "element-interface @ git+https://github.com/datajoint/element-interface.git",
-        "facemap @ git+https://github.com/kushalbakshi/facemap.git",
+        "facemap[gui] @ git+https://github.com/datajoint/facemap.git",
```

Comment:
Following other elements, such as [`element-deeplabcut` setup](https://github.com/datajoint/element-deeplabcut/blob/main/setup.py), I would move the installation of the GUI to `extras_require` and make it optional.